core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -929,7 +929,7 @@
   /**
    * default value for local dictionary generation
    */
-  public static final String LOCAL_DICTIONARY_ENABLE_DEFAULT = "true";
```

Comment:
Please correct the documentation also as default value as false

Reply:
ok